src/js/helpers/disasterHelper.js

```diff
@@ -116,3 +116,7 @@ export const fetchLoanSpending = (type, params) => apiRequest({
     method: 'post',
     data: params
 });
+
+export const fetchCovidTotals = () => apiRequest({
+    url: 'v2/disaster/overview/?def_codes=L,M,N,O,P'
```

Comment:
We have an API for the def codes https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/references/def_codes.md

There is also a helper in the `disasterHelper` to create this query string `defCodeQueryString`.